packages/toolkit/utils/src/index.ts

```diff
@@ -1,44 +1,3 @@
 export * from './compiled';
-export * from './commands';
-export * from './FileSizeReporter';
-export * from './printBuildError';
-export * from './debug';
-export * from './findExists';
-export * from './is';
-export * from './compatRequire';
-export * from './logger';
-export * from './constants';
-export * from './ensureArray';
-export * from './ensureAbsolutePath';
-export * from './clearConsole';
-export * from './applyOptionsChain';
-export * from './getBrowserslist';
-export * from './removeSlash';
-export * from './getEntryOptions';
-export * from './getPort';
-export * from './monorepo';
-export * from './getPackageManager';
-export * from './runtimeExports';
-export * from './readTsConfig';
-export * from './path';
-export * from './pathSerializer';
-export * from './generateMetaTags';
-export * from './prettyInstructions';
-export * from './alias';
+export * from './cli';
```

Comment:
What is the difference between "cli" and "runtime-node"?

"cli" code is also running in Node.js environment.

Reply:
The "cli" uses CJS, and the "runtime-node" uses ESM.  

- One reason for the separation is treeshaking.
- Second is to give a clear directory division to avoid unexpected packaging when adding new functions or function add new 'import xxx'

Reply:
We put the code that both cli and runtime used into universal, and only the code here needs to pay attention to the problem of side effects.

Reply:
get👌🏻